src/identity/apis/auth.ts

```diff
@@ -227,6 +228,25 @@ export const fetchAccountDetails = async (
   return accountDetails
 }
 
+// change the user's default account
+export const putDefaultQuartzAccount = async (
+  accountId: number
+): Promise<Organization> => {
+  const response = await putAccountsDefault({
+    data: {
+      id: accountId,
+    },
+  })
+
+  if (response.status === 500) {
+    throw new ServerError(response.data.message)
+  }
+
+  // success status code is 204; no response in body is expected.
+  const responseEmpty = response.data
+  return responseEmpty
```

Comment:
I think this should be removed, since it doesn't return anything and this is kind of confusing, and it looks like the return value isn't used by anything anyway. The return value should probably be `Promise<void>`. JavaScript functions implicity return `undefined` if there's no return value, which it looks like will be the same as this code. The comment should probably stay, just to keep someone who is reading this block of code from having to look at the definition of `putAccountsDefault` in the `quartzRoutes`